.github/workflows/release-angular.yml

```diff
@@ -0,0 +1,87 @@
+name: Release @scania/tegel-angular
+on:
+  workflow_dispatch:
+    inputs:
+      tegelVersion:
+        description: 'Tegel version'
+        required: true
+        type: string
+
+      nodeVersion:
+        description: 'Node version'
+        required: true
+        default: '18.16.0'
+        type: string
+
+      tags:
+        description: 'Tag'
+        required: true
+        default: 'beta'
+        type: choice
+        options:
+          - latest
+          - beta
+          - dev
+
+      dryRun:
+        description: 'Dry run'
+        required: false
+        default: true
+        type: boolean
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        # Setup .npmrc file to publish to npm
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ github.event.inputs.tegelVersion }}
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Angular - CD into correct folder
+        run: echo "cd packages/angular"
+
+      - name: Angular - Remove package-lock
+        run: echo "rm -rf package-lock.json"
+
+      - name: Angular - Install latest tegel package
+        run: npm install @scania/tegel@${{ github.event.inputs.tegelVersion }}
+
+      - name: Set user
+        run: git config --global user.name "Tegel - Scania" && git config --global user.email "tegel.design.system@gmail.com"
+
+      - name: Angular - Bump version
+        run: npm version ${{ github.event.inputs.tegelVersion }}
+
+      - name: Angular - Read Package.json Version
+        id: version
+        run: echo "version=$(jq -r .version package.json)" >> $GITHUB_OUTPUT
+
+      - name: Check out new branch
+        run: git checkout -b "release/${{ steps.version.outputs.version }}"
+
+      - name: Angular - Run build
+        run: npm run build
```

Comment:
I think it would be better if the build step was before the "checkout and create branch" step, so no branch is created in case the build fails.
Might not be an issue, but also might be

Reply:
True!